### PR TITLE
New changes for reviewing

### PR DESCRIPTION
**Pull request by IlaganJireh**

### CODE REVIEW

File: ilagan_andreijireh/script.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+function validateInput() {
+const inputField = document.getElementById("comment");
+const nameField = document.getElementById("name");
+const submitBtn = document.getElementById("submit_btn");
+
+const isValid = (field) => field.value.trim().length > 0;
```
Comment (Naoe-Adrian) [inline ilagan_andreijireh/script.js]: use guardclause here it can be like this
```suggestion
const isValid = (field) => !field.value.trim().length;
```

**Review by Naoe-Adrian — CHANGES_REQUESTED** [review]
lgtm also add another guard clauses

**Comment by IlaganJireh** [comment]
I already changed my codes according to your review, thank you!

**Review by Naoe-Adrian — APPROVED** [review]
LGTM thank you

**Pull request merged**